Cubical/Functions/FunExtEquiv.agda

```diff
@@ -31,6 +31,8 @@ module _ {A : Type ℓ} {B : A → Type ℓ₁} {f g : (x : A) → B x} where
   funExtPath : (∀ x → f x ≡ g x) ≡ (f ≡ g)
   funExtPath = ua funExtEquiv
 
+funExtIso : ∀ {ℓ ℓ₁} {A : Type ℓ} {B : A → Type ℓ₁} {f g : (x : A) → B x} → Iso (∀ x → f x ≡ g x) (f ≡ g)
```

Comment:
Why reprove funExt-iso here? We should only have one of them and I have no strong opinions about which

Reply:
It might fit in better here, since this is where the rest of the `funExt` properties are stated. I will remove it from the Isomorphism file.